src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -203,6 +203,8 @@ protected virtual void Initialize()
                 new CSharpFunctionDescriptionProvider(this, ScriptConfig)
             };
 
+            NodeFunctionInvoker.UnhandledException += OnUnhandledException;
```

Comment:
Not terribly happy about this and will revisit to see if there is a more generalized way of doing this